src/index.js

```diff
@@ -8,6 +8,16 @@ const config = require('./data/client.json')
 const updateServer = require('./UpdateServer.js')
 const Util = require('./Util.js')
 
+async function updateBlacklists () {
+  const response = await request(`https://discord.com/api/v8/guilds/${config.banServer}/bans`, { json: true, headers: { Authorization: `Bot ${config.token}` }, resolveWithFullResponse: true, simple: false })
+  let bans = {}
+  if (response.statusCode === 200) response.body.forEach(ban => { bans[ban.user.id] = true })
+  return bans
+}
+
+const blacklists = Promise.resolve(updateBlacklists())
+module.exports = blacklists
```

Comment:
This doesn't work. The shards run in an entirely different process, so importing index.js will run this file brand new every time. In fact, I think this code as it exists would cause an infinite loop of the master process starting a shard, which requires the master code, which starts a shard.. etc, onwards forever.

Instead of exporting, you should just load the blacklist data into the global Cache before launching shards below.